packages-kintone-customize/app-292-grossProfitStatement/src/eventHandlers/helpers/roundDownTo1000.ts

```diff
@@ -0,0 +1,12 @@
+import { Big } from 'big.js';
+
+
+/** 1000未満を切り捨てる */
+export const roundDownTo1000 = (amount: number) => {
+  const bigAmount = new Big(amount);
+  const roundedAmount = bigAmount.div(1000)
+    .round(0, 0)
+    .times(1000);
```

Comment:
```suggestion
  const roundedAmount = bigAmount
    .round(-3, Big.roundDown);
```

[ドキュメンテーション](https://mikemcl.github.io/big.js/#round)による、roundだけで、切り捨ては可能です。
テストに合格していますが、ご確認ください。

Reply:
ご指摘ありがとうございます。
ドキュメンテーションの見る場所を間違えていて、dpにはマイナスが入れれないものと思い込んでしまっていました。
上記の通り修正しました。
シンプルになったので、関数化の必要もないかな？と思いましたが、別DBとのデータ統合の際に検討することとして、今はこのまま残しておきます。